targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_GR_LYCHEE/TARGET_MBED_MBRZA1LU/reserved_pins.h

```diff
@@ -0,0 +1,6 @@
+#ifndef RESERVED_PINS_H
```

Comment:
Should this contain license header on top?

Reply:
I modified this content in commit 4bc79b0.